src/openrct2/network/network.cpp

```diff
@@ -264,7 +264,7 @@ bool Network::BeginClient(const char* host, uint16 port)
 	return true;
 }
 
-bool Network::BeginServer(uint16 port, const char* address)
+bool Network::BeginServer(uint16 port)
```

Comment:
Keep this parameter. Its better to pass in `gConfigNetwork.listen_address` from the caller.